service/src/main/java/uk/gov/hmcts/reform/fpl/utils/ElementUtils.java

```diff
@@ -82,6 +82,12 @@ private ElementUtils() {
     public static <T> DynamicList asDynamicList(List<Element<T>> elements,
                                                 UUID selectedId,
                                                 Function<T, String> labelProducer) {
+        return asDynamicList(emptyList(), elements, selectedId, labelProducer);
+    }
+
+    public static <T> DynamicList asDynamicList(List<DynamicListElement> additionalItems, List<Element<T>> elements,
```

Comment:
just noticed something about this file, the wrap elements methods do not set an id, they are always null.

Reply:
i think it may be done on purpose. If id is not provided ccd will generate it. Having null id simplify tests, as it is easier to match elements